dashboard/src/shared/PackageRepositoriesService.ts

```diff
@@ -282,4 +292,84 @@ export class PackageRepositoriesService {
         throw convertGrpcAuthError(e);
       });
   }
+
+  public static getRepositoriesPermissions = (
+    namespace: string,
+    config: IConfig,
+    plugin: Plugin,
+  ): IPackageRepositoryPermission | undefined => {
+    switch (plugin.name) {
```

Comment:
I think the answer is no... but: can we obtain these strings from the plugin backend somehow?  Like, what if `packaging.carvel.dev` get eventually renamed?

Reply:
There is a comment exactly about that a little bit below.
We could even add a new API endpoint to provide the permissions of the user for e.g. repos.
In this way the UI would not need to know anything about resource group, name, verb, etc.